### DISCUSSION TITLE
Getting warnings when building

### DISCUSSION THREAD

**Discussion by benjamin-9T** [Live-Charts/LiveCharts2#587] [Q&A]
When I build using LiveChartsCore.SkiaSharpView.WPF (350) I get the following Warning messages:

Warning		The referenced component '..\.nuget\packages\skiasharp.nativeassets.macos\2.88.1-preview.91\buildTransitive\net462\..\..\runtimes\osx\native\libSkiaSharp.dylib' could not be found.	
Warning		The referenced component '..\.nuget\packages\harfbuzzsharp.nativeassets.macos\2.8.2.1-preview.91\buildTransitive\net462\..\..\runtimes\osx\native\libHarfBuzzSharp.dylib' could not be found.	

These come from OS X and since I am compiling in Visual Studio c# Framework 4.7.2 I do not really need that :)

Is there a way to fix this issue?

**Comment by alexachso** [comment]
I have logged the same question here https://github.com/beto-rodriguez/LiveCharts2/discussions/428. Did not received a response yet :(

**Comment by beto-rodriguez** [comment]
Sorry for the late response, but the truth is is that I am not completely sure, this seems an SkiaSharp issue, I am afraid I have no power to change this.

**Reply by alexachso** [reply]
Thank you Alberto for the reply! That is ok, I was just wondering if I did something wrong or if everybody has the same warnings. Maybe the SkiaSharp team will resolve this in the future :)

**Comment by mayank07072019** [comment]
Any solution on this , I am also getting the same issue.
